helm/orca/config/alerts-mapping.yaml

```diff
@@ -933,19 +897,110 @@ zabbix:
         kind: node
         properties:
           name: node
-    - name: "Temperature is above warning threshold: >{$TEMP_WARN:}"
+  regex_mappings:
```

Comment:
From the CRD perspective, it will be better to make the mapping type more granular. Let's specify the type for each mapping entry instead of providing two separate lists for plain and regex-based mappings:

```yaml
    - name: ".*Overall System Status Error.*"
      type: regex
      source_mapping:
        origin: kubernetes
        kind: node
        properties:
          name: node
    - name: ".*RAID Controller Error.*"
      type: regex
      source_mapping:
        origin: kubernetes
        kind: node
        properties:
          name: node
```